people.go

```diff
@@ -339,10 +341,43 @@ func getPersonByPubkey(w http.ResponseWriter, r *http.Request) {
 func getPersonByUuid(w http.ResponseWriter, r *http.Request) {
 	uuid := chi.URLParam(r, "uuid")
 	person := DB.getPersonByUuid(uuid)
+	assetBalanceData, err := getAssetByPubkey(person.OwnerPubKey)
+
+	personResponse := make(map[string]interface{})
+	personResponse["id"] = person.ID
+	personResponse["uuid"] = person.Uuid
+	personResponse["owner_alias"] = person.OwnerAlias
+	personResponse["unique_name"] = person.UniqueName
+	personResponse["description"] = person.Description
+	personResponse["tags"] = person.Tags
+	personResponse["img"] = person.Img
+	personResponse["created"] = person.Created
+	personResponse["updated"] = person.Updated
+	personResponse["unlisted"] = person.Unlisted
+	personResponse["deleted"] = person.Deleted
+	personResponse["last_login"] = person.LastLogin
+	personResponse["owner_route_hint"] = person.OwnerRouteHint
+	personResponse["owner_contact_key"] = person.OwnerContactKey
+	personResponse["price_to_meet"] = person.PriceToMeet
+	personResponse["extras"] = person.Extras
+	personResponse["twitter_confirmed"] = person.TwitterConfirmed
+	personResponse["github_issues"] = person.GithubIssues
+	personResponse["new_ticket_time"] = person.NewTicketTime
```

Comment:
is there any easier way to assign all of these ik in javascript you can do `let thing = { ...person }` 

I'm not as familiar with golang

Reply:
dont think so @kevkevinpal 

@omoniyi24 we do not need `created`, `updated`, `unlisted`, `deleted`, `last_login`, or `new_ticket_time`